src/main/java/Duke.java

```diff
@@ -1,10 +1,59 @@
+import duke.*;
```

Comment:
According to the coding standards, imported classes should always be listed explicitly.